tests/testthat/test-emissions_profile_upstream.R

```diff
@@ -181,7 +181,7 @@ test_that("at product level, `NA` in a benchmark yields `NA`s only in the corres
   expect_false(is.na(filter(out, clustered == "a")$risk_category))
   expect_true(is.na(filter(out, clustered == "b")$risk_category))
 
-  benchmark <- "input_tilt_sector"
+  benchmark <- "input_tilt_subsector"
   co2 <- example_inputs(
```

Comment:
@Tilmon @AnneSchoenauer I have also changed the benchmark from `input_tilt_sector` to `input_tilt_subsector`. This change will affect the `emissions_profile_upstream` indicator. I have done this change to remain aligned with the methodology we are using for `emission_profile` indicator. Should I do this change or not?

Reply:
Yes, it should also be for the emission_profile_upstream. However, as we don't estimate the upstream yet for the Bundesbank and publishing the data on our webtool it is fine if you just write a reminder ticket and do this at a later stage! Best Anne

Reply:
Hi Kalash - okay! Let's do it like this :) Please also clarify a bit more what you mean by "estimation" :)

Reply:
Hi @AnneSchoenauer, by estimation I mean any kind of analytical assessment that you and Tilman might have to do for `emission_profile_upstream` if "input_tilt_sector" is changed to "input_tilt_subsector". I am not sure if its required. If its not required then I will not create any ticket as a reminder. Please conform if you guyz need a ticket!

Reply:
I think its fine without a ticket :) Thanks!